lib/meilisearch/error.rb

```diff
@@ -1,45 +1,58 @@
 # frozen_string_literal: true
 
 module MeiliSearch
-  class MeiliSearchError < StandardError; end
-  class IndexUidError < MeiliSearchError; end
-  class MeiliSearchTimeoutError < MeiliSearchError
-    attr_reader :message
+  class ApiError < StandardError
```

Comment:
```suggestion
  class ServerError < StandardError
```

I am not 100% sure about this, but maybe we could improve clarity by talking about the MeilISearch server instead of the API. Having `API` in the code is. imho, one of the most common term that could refer to anything. In our usage of this wrapper, I think `Server` instead of `API` might be a little more explicit. WDYT?

Reply:
This naming was chosen here https://github.com/meilisearch/integration-guides/issues/19 after we discussed together in meeting 😅 It would imply to change it in all the SDKs, but it's still possible. If you strongly think the name should be changed, you can write down your suggestion in the issue I link 🙂